### ISSUE TITLE
Log File owner/group should default to be the same as the watcher's

### ISSUE THREAD

**Issue by ayashjorden** [circus-tent/circus#990]
Hi,
When using the FileStream/WatchedFileStream classes, the file gets created with the owner/group of the circusd process.

This can require extra logic from the user when interacting with the log files (move/delete/read operations.)
It also conflicts when the log files are stored in the application log directory where the other files are owned by the user/group that run the application.

Is it possible to configure the files to get created with the owner/group of the watcher?

Thank you,
Yarden

**Comment by k4nar** [comment]
Indeed, this could be an additional option for those stream handlers.
